keylime/ima/ima_dm.py

```diff
@@ -293,7 +294,7 @@ def validate_table_load(self, event: "LoadEvent", match_key, digest) -> Failure:
         # Check "num_targets"
         # Note that we get actually could get multiple lines, but this does not happen for our use cases so we
         # treat it as a failure.
-        if event.device_metadata.num_targets != len(event.targets) or used_policy["table_load"][entry] != len(
+        if event.device_metadata.num_targets != len(event.targets) or used_policy["table_load"]["num_targets"] != len(
```

Comment:
Probably was assuming that entry leaves the loop above with "num_targets". Tricky. LGTM.